zerver/openapi/zulip.yaml

```diff
@@ -8474,6 +8474,233 @@ paths:
                               type: string
                               description: |
                                 The text content of the status message.
+                      user_settings:
+                        type: object
+                        description: |
+                          Present if `user_settings` is present in `fetch_event_types`.
+
+                          A dictionary which contains the settings of a user.
+
+                          **Changes**: New in Zulip 5.0 (feature level 89).
```

Comment:
This should have a 

"Previously, these settings appeared in the top-level object, where they are available
for clients without the `user_settings_object` client capability for backwards-compatibility.

Reply:
Done.